xep-0068.xml

```diff
@@ -171,7 +177,7 @@
     </section2>
     <section2 topic='Incorrectly Specified FORM_TYPE' anchor='usecases-incorrect'>
 
-    <p>If the FORM_TYPE field is not hidden, it MUST be ignored as a context indicator.</p>
+    <p>If the FORM_TYPE field is not hidden in a form with type=&quot;form&quot; or type=&quot;result&quot;, it MUST be ignored as a context indicator.</p>
```

Comment:
Maybe elaborate on the consequence of this: "As consequence, FORM_TYPE fields MAY omit the hidden attribute for 'submit' type forms."